spec/v0-1.md

```diff
@@ -111,6 +118,14 @@ The expression `(eval <exp>)` or `(eval <exp> <env>)` first evaluates the expres
 
 While `eval` is part of core Lurk, it is not part of the initial `lurk-rs` release.
```

Comment:
Should this note be kept in then?

Reply:
We should probably just remove `eval` completely. We can add it back when actually supported in `lurk-rs`.